operator/v1/types_ingress.go

```diff
@@ -543,6 +545,7 @@ type AWSLoadBalancerParameters struct {
 	// networkLoadBalancerParameters holds configuration parameters for an AWS
 	// network load balancer. Present only if type is NLB.
 	//
+	// +openshift:enable:FeatureGate=SetEIPForNLBIngressController
```

Comment:
Has this been discussed elsewhere? We are taking an existing field out of GA and back into TechPreview? That's interesting.

I know there isn't any fields under `newtorkLoadBalancer`, however, that doesn't mean you could have an empty struct `networkLoadBalancer: {}` in your IngressController CRD.

I don't think this is a compatible change.

Reply:
https://github.com/openshift/api/pull/1826#discussion_r1571239069 @deads2k suggested on the field eipAllocations.
https://github.com/openshift/api/pull/1826#discussion_r1650777052 @JoelSpeed suggested above networkLoadBalancer

Reply:
But this is in a different file, on the IngressController. Didn't they mean for the Ingress object? Their comments were on `config/v1/types_ingress.go`.

Reply:
@gcs278 sure I have just made the changes related to this on the config/v1 only as we already had networkLoadBalancer in GA